doc/source/analyzing/fields.rst

```diff
@@ -604,26 +607,26 @@ as detailed below (see :ref:`field_parameters` for more information).
 
 The relative fields which are currently supported for gas fields are:
 
-* ``("gas", "relative_velocity_{xyz}")``, defined by setting the
-  ``"bulk_velocity"`` field parameter
-* ``("gas", "relative_magnetic_field_{xyz}")``, defined by setting the
-  ``"bulk_magnetic_field"`` field parameter
+* ``('gas', 'relative_velocity_x')``, defined by setting the
+  ``'bulk_velocity'`` field parameter
+* ``('gas', 'relative_magnetic_field_x')``, defined by setting the
+  ``'bulk_magnetic_field'`` field parameter
 
```

Comment:
I liked how the old lines 607-609 implied that the fields were defined for each component. I understand why you changed it, but it'd be nice to note it in some other, more explicit way. Maybe just add the following note?

```suggestion

Note that fields ending in ``'_x'`` are defined for each component.

```
```